wagtail/wagtailsnippets/tests.py

```diff
@@ -131,6 +131,19 @@ def test_simple(self):
         response = self.get()
         self.assertEqual(response.status_code, 200)
         self.assertTemplateUsed(response, 'wagtailsnippets/snippets/create.html')
+        self.assertNotContains(response, '<ul class="tab-nav merged">')
```

Comment:
You're right, that does not work in the way I hoped it would. Ideally there would be a way to check that there is/is not a specific node in some HTML, while ignoring the contents of the node. That is what I hoped having an unclosed `<ul>` tag would do, but I was wrong. Don't worry about this test then.